GrandCentralBoard/App/ViewControllers/LoadingViewController.swift

```diff
@@ -18,7 +18,8 @@ class LoadingViewController: UIViewController {
         BonusWidgetBuilder(dataDownloader: self.dataDownloader),
         GoogleCalendarWatchWidgetBuilder(),
         HarvestWidgetBuilder(),
-        ImageWidgetBuilder(dataDownloader: self.dataDownloader)
+        ImageWidgetBuilder(dataDownloader: self.dataDownloader),
+        WebsiteAnalyticsWidgetBuilder()
```

Comment:
This widget covers very narrow scope of website analytics.

`BlogPostsPopularityWidget` perhaps?